www/artists/arocamora/vessels.html

```diff
@@ -4,77 +4,104 @@
+                        <th>
+                            cactus blossom bottle: porcelain sculpture, under glazes, 1997
+                        </th>
+                        <th>
+                            porcelain with underglazes
+                        </th>
+                        <th>
+                            porcelain with underglazes
+                        </th>
+                    </tr>
+                    <tr>
+                        <td>
+                            <div class="c1">
+                                <img src="images/blurry1.jpg" alt="Anita Rocamora" width="200" height="200">
+                            </div>
+                        </td>
+                        <td>
+                            <div class="c1">
+                                <img src="images/blurry2.jpg" alt="Anita Rocamora" width="200" height="200">
+                            </div>
+                        </td>
+                        <td>
+                            <div class="c1">
+                                <img src="images/slit1.jpg" alt="Anita Rocamora" width="200" height="200">
+                            </div>
+                        </td>
+                    </tr>
+                </table>
                 <br>
-                GIVE & RECEIVE I & II: porcelain sculpture, under glazes, 2007
-            </p>
-            <h2>
                 Vessels are hollow utensil, such as a cup, vase, or pitcher, used as a container, especially for liquids
-            </h2>
-            <p>
-                <a href="images/vessel04.jpg"><img src="images/vessel04.jpg" alt="vessel"></a>
                 <br>
-                cactus blossom bottle: porcelain sculpture, under glazes, 1997
-            </p>
-            <p>
-                <a href="images/vessel02.jpg"><img src="images/vessel02.jpg" alt="vessel"></a>
                 <br>
-                VESSEL I: porcelain with underglazes
-            </p>
-            <p>
-                <a href="images/vessel01.jpg"><img src="images/vessel01.jpg" alt="Vessel"></a>
+                Anita Rocamora is visual artist who works in ceramics and mixed media. Anita uses natuer and humanity for insperation in her art work. Anita uses deferent shapes from natuer and put them in a shape for a vase. Anita trys to create objects that are entirely imaginary yet evoke a feeling of familiarity and recognition. In nature. Form and function flow from one another in even the most absurdly extravagant creations. The focus of her work is to exploration other shapes that no one ever used. Anitas primary materials are porcelian and stoneware. Clay contains traces of all that has existed on this planet. From single-cell organisms to mountains. To ground together by erosion and geological forces to create a material that can be transformed at will. Anita likes to make with these material objects for facilitate joy, awareness, whimsy, and a sense of connection to the earth.
```

Comment:
And you've got it again here? Seriously, did you just blindly copy/paste the format from index.html, without even paying attention to what you were doing?